inicheck/cli.py

```diff
@@ -13,7 +12,18 @@
 
 
 def current_version():
-    return get_version(root='..', relative_to=__file__)
+    if sys.version_info >= (3, 8):
+        from importlib.metadata import version, PackageNotFoundError
+        try:
+            return version('inicheck')
+        except PackageNotFoundError:
+            'unknown'
+    else:
+        from pkg_resources import get_distribution, DistributionNotFound
```

Comment:
To keep things simpler, I think you can just use pkg_resources. This is what I did in SMRF to get around the python versions that don't have importlib. But as it is now works as well and will be good to see how it works during a pip install.

Reply:
Good point. Didn't check thoroughly enough that `importlib` was a recent 3.8 introduction. At the very least a good reference for 3.8+ version support.